token/js/client/token.js

```diff
@@ -139,6 +145,11 @@ export const MintLayout: typeof BufferLayout.Structure = BufferLayout.struct([
  * Information about an account
  */
 type AccountInfo = {|
+  /**
+   * The address of this account
+   */
+  address: PublicKey,
```

Comment:
I needed this. Othwerwise, `getOrCreateAssociatedAccountInfo` is too cumbersome to use.